src/pages/ProductList/components/Product/index.js

```diff
@@ -3,35 +3,45 @@ import React from 'react'
 import urls from '../../../../constants/urls'
 import {
   Wrapper,
+  Inner,
   ImgWrap,
   Img,
   TitleWrap,
   Title,
   Price,
   Link,
-  AddButton,
 } from './styled'
+import { ButtonGroup } from '../../../../components/Typography'
+import { Button } from '../../../../components/Button'
+import { toast } from 'react-toastify'
 
 const Product = ({ node, addProduct }) => (
-  <Wrapper>
-    <Link to={urls.productDetail(node.id)}>
+  <Wrapper data-testid="product-in-list">
+    <Inner>
       <ImgWrap>
-        <Img src={node.image_url} alt={`${node.name} image`} />
+        <Link to={urls.productDetail(node.id)}>
+          <Img src={node.image_url} alt={`${node.name} image`} />
+        </Link>
       </ImgWrap>
       <TitleWrap>
         <Title>{node.name}</Title>
       </TitleWrap>
       <Price>{node.price.formatted_amount}</Price>
-      <AddButton
-        type={'button'}
-        onClick={evt => {
-          evt.preventDefault()
-          addProduct(node.id)
-        }}
-      >
-        Add to Cart
-      </AddButton>
-    </Link>
+      <ButtonGroup>
+        <Button
+          onClick={evt => {
+            evt.preventDefault()
```

Comment:
Seems to be unnecessary to prevent event here

Reply:
Fixed!